root/utility/releaseGroupType.js

```diff
@@ -7,7 +7,7 @@
  * later version: http://www.gnu.org/licenses/gpl-2.0.txt
  */
 
-import {l_attributes} from '../static/scripts/common/i18n/attributes';
+import {lp_attributes} from '../static/scripts/common/i18n/attributes';
```

Comment:
This file ended up not being used in the implementation, but this was broken anyway, so. I guess it could be split into a different commit if it matters :)

Reply:
If you don't mind. :P It's just a bit confusing when you're looking at history later or using git bisect to find out what change introduced a bug.